snapcraft/internal/project_loader/_extensions/gnome_3_38.py

```diff
@@ -140,6 +140,9 @@ def __init__(self, *, extension_name: str, yaml_data: Dict[str, Any]) -> None:
                 {
                     "GDK_PIXBUF_MODULE_FILE": "/snap/gnome-3-38-2004-sdk/current/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/gdk-pixbuf-current/loaders.cache"
                 },
+                {
+                    "ACLOCAL_PATH": "/snap/gnome-3-38-2004-sdk/current/usr/share/aclocal:$ACLOCAL_PATH"
```

Comment:
To prevent expansion of ACLOCAL_PATH is undefined:
```suggestion
                    "ACLOCAL_PATH": "/snap/gnome-3-38-2004-sdk/current/usr/share/aclocal{ACLOCAL_PATH:+:$ACLOCAL_PATH}"
```